homeassistant/components/binary_sensor/rainmachine.py

```diff
@@ -50,6 +50,21 @@ def __init__(self, rainmachine, sensor_type, name, icon):
         self._sensor_type = sensor_type
         self._state = None
 
+    @property
+    def device_info(self):
```

Comment:
Why not put this in `RainMachineEntity` in the component module? It seems to be the same for all the platforms.